### PR TITLE
Log the ProposalState enum more nicely.

### PR DESCRIPTION
**Pull request by pphaneuf**
This allows using CHECK_EQ, which shows the values (quite useful when debugging!).

### CODE REVIEW

**Comment by pphaneuf** [comment]
The Travis failure should be fixed when #387 is merged.

File: cpp/util/masterelection.cc [hunk 1]
```diff
@@ -43,6 +43,28 @@ string EnsureEndsWithSlash(const string& s) {
 }  // namespace
 
 
+std::ostream& operator<<(std::ostream& output,
+                         MasterElection::ProposalState state) {
+  switch (state) {
+#define MY_PRINT_OUT_PROPOSALSTATE(state)    \
```
Comment (AlCutter) [inline cpp/util/masterelection.cc]: ewww

Comment (pphaneuf) [inline cpp/util/masterelection.cc]: Want me to repeat it "out long"? I guess putting it on one line should make it "not too bad", but it's repetitive (even within a line)...

Comment (AlCutter) [inline cpp/util/masterelection.cc]: Yes please :)

**Comment by pphaneuf** [comment]
Addressed the comment.

**Comment by AlCutter** [comment]
LGTM

**Pull request merged**